.github/workflows/common.yml

```diff
@@ -87,7 +87,8 @@ jobs:
           go_version: ${{ env.GO_VERSION }}
           reporter: github-pr-review
           fail_on_error: true
-          golangci_lint_flags: "-c=.golangci-required.yml --out-format=line-number"
+          cache: false
```

Comment:
There is some problems with this option. It seems that it conflicts with our other caches. Disabled for now.